### DISCUSSION TITLE
ficheiros SaF-T Rejeitados

### DISCUSSION THREAD

**Discussion by inmx** [assoft-portugal/SAF-T-AO#167] [General]
Saudação pessoal, alguns clientes nossos têm os seus ficheiros SAF-T rejeitados por parte do validador da AGT, deste modo gostaria de saber quais são os motivos que estão na base da rejeição do ficheiro SAF-T  visto que se encontram nos conformes?

Alguém poderia me ajudar

**Comment by cryptolopes** [comment]
Não é um Issue relacionado com o SAF-T.
Usa o [Discussions](https://github.com/assoft-portugal/SAF-T-AO/discussions) para este tipo de dúvidas.
